src/tiled.lisp

```diff
@@ -358,6 +393,86 @@
         (gid (tobject-gid tobject))
         (image (tobject-image tobject)))
     (cond
+      (template
+       (let* ((template-object (load-template template))
```

Comment:
This line should be updated to `(%load-template template #'read-file-into-string)` after updating `load-template`